DevelopmentProposals/template.md

```diff
@@ -0,0 +1,63 @@
+## Project: TITLE
+***OpenSim Design Document***
+
+### Overview
+
+Just a few sentences that describes the purpose of this project. For example:
+Create a graphical tool to plot simulation data.
```

Comment:
We should probably revise the example throughout since proposals for the GUI and plotting are not the purview of the opensim-core repo. Just to avoid any confusion.